### PR TITLE
Robert border assignment 2

### PR DESCRIPTION
**Pull request by Robert-Border**
Hey @richardking 

Created the 3 apps for assignment 2 and tried to do the bonus items where possible. Apologizing in advance for what is probably a lot of repetitive parts of the code that could have been made more DRY.

### CODE REVIEW

File: assignment-2-ruby/rborder/tiny_instacart.rb [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+# select a store
+def select_warehouse
+  selected_store = ""
+  stores = ['Kroger','Publix','Aldi','Sprouts']
+  stores.each do |store|
+    print "| #{store} |"
+  end
+  puts "\n"
+  while stores.include?(selected_store) == false do
```
Comment (richardking) [inline assignment-2-ruby/rborder/tiny_instacart.rb]: could do `while !stores.include?(selected_store) do`

File: assignment-2-ruby/rborder/tiny_instacart.rb [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    print "\nSelect an available store: "
+    selected_store = gets.chomp
+  end
+  return selected_store
```
Comment (richardking) [inline assignment-2-ruby/rborder/tiny_instacart.rb]: fwiw ruby automatically returns the last line, so don't need to say `return selected_store`, just `selected_store`

File: assignment-2-ruby/rborder/order_items_data.rb [hunk 3]
```diff
@@ -0,0 +1,56 @@
+AISLES = ['Produce','Dairy & Eggs','Meat & Seafood','Snacks','Deli','Frozen Foods','Bulk','Household','Bakery']
+
+ITEMS = [{name: "Lettuce",  aisle: "Produce",   cost: 1},
```
Comment (richardking) [inline assignment-2-ruby/rborder/order_items_data.rb:3]: could look into creating and ingesting a csv/yaml file for this type of structured data

**Review by richardking — APPROVED** [review]

**Comment by richardking** [comment]
overall looks great, left a few nit/stylistic comments. have you guys started learning about classes/instances etc yet?

File: assignment-2-ruby/rborder/tiny_instacart.rb (hunk 1 above)
Comment (Robert-Border) [reply assignment-2-ruby/rborder/tiny_instacart.rb]: Ah! Bang operators. May take a little to get the muscle memory there.

File: assignment-2-ruby/rborder/order_items_data.rb (hunk 3 above)
Comment (Robert-Border) [reply assignment-2-ruby/rborder/order_items_data.rb:3]: I think YAML files are coming up, but I didn't want to jump too far ahead.

**Review by Robert-Border — COMMENTED** [review]
We have started learning about classes, but I think we are going to have a homework using OOP coming up.

**Pull request merged**